audbcards/core/dataset.py

```diff
@@ -144,6 +144,35 @@ def languages(self) -> typing.List[str]:
         r"""Languages of the database."""
         return self.header.languages
 
+    @property
+    def iso_languages(self) -> typing.List[str]:
+        r"""Languages of the database."""
+        return self._map_iso_languages(self.languages)
+
+    @staticmethod
+    def _map_iso_languages(languages : typing.List[str]) -> typing.List[str]:
+        r"""Calculate ISO languages for a list of languages.
+
+        Args:
+            languages: list of languages as given in the header languages
+        Returns:
+            list of languages
+
+        Behavior: leaves languages intact if `audformat.utils.map_language`
+        raises ValueError
```

Comment:
Usually, some extra text is placed between the first single line and the arguments.

```suggestion
        Leaves languages intact if :func:`audformat.utils.map_language`
        raises :exception:`ValueError`.
        
        Args:
            languages: list of languages as given in the header languages
            
        Returns:
            list of languages
```